src/components/shared/DateAndDebit/DateAndDebit.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import styles from "./DateAndDebit.module.scss";
+import format from "date-fns/format";
+
+export function DateAndDebit({ date, debit, currencySymbol }) {
+  const givenDate = format(date, "d, MMMM yyyy");
+
+  return (
+    <div className={styles.layoutDateAndDebit}>
+      <div className={styles.date}>{givenDate}</div>
+      <div className={styles.debit}>
+        {" "}
```

Comment:
Don't use JSX spacing here. Instead, if you need spacing, wrap the text you want spacing for in a `span`, and add margins to the span. If this is Prettier, then just delete the `{" " }`.